src/planscape/eval/compute_conditions.py

```diff
@@ -91,11 +123,15 @@ def score_metric(condition_reader: ConditionReader, metric: Metric, condition_ty
     """
     if not 'filepath' in metric:
         return None
-    return condition_reader.read(metric['filepath'], condition_type) if metric['filepath'] else None
+    condition = condition_reader.read(metric['filepath'], condition_type)
+    if metric['filepath']:
+        return condition
+    else:
```

Comment:
Having trouble resolving the types: new_profile seems to be a dictionary, but RasterData seems to want a str for its second argument. Can you clarify?